### PR TITLE
fix the buggy preview screen

### PR DESCRIPTION
**Pull request by karissa**
https://www.dropbox.com/s/gbt97yucl2ekmp4/magic.mov?dl=0

the root element for each item needed to be outside of `module.exports` so it never changes, but gets updated by the `render-data` module

### CODE REVIEW

**Comment by karissa** [comment]
@Kriesse when a file can't be rendered..
screen shot 2016-09-13 at 1 50 22 pm

File: client/js/models/preview.js [hunk 1]
```diff
@@ -29,11 +25,7 @@ module.exports = {
   effects: {
     file: (data, state, send, done) => {
       send('preview:update', data, noop)
-      send('preview:openPanel', {}, function () {
-        send('archive:readFile', data, function (readStream) {
-          send('preview:update', {readStream: readStream}, noop)
-        })
-      })
+      send('preview:openPanel', {}, done)
```
Comment (laurengarcia) [inline client/js/models/preview.js]: yeah, i was going to suggest this actually. it is cleaner.

**Comment by laurengarcia** [comment]
lgtm but i haven't pulled this down and tried locally

**Comment by karissa** [comment]
fixes #240

**Pull request merged**